asv_bench/asv.conf.omnisci.json

```diff
@@ -72,9 +72,9 @@
     // the package name by 'pip+', and the package will be installed via
     // pip (with all the conda available packages installed first,
     // followed by the pip installed packages).
-    "matrix": {
-        "omniscidbe4py": "",
-    },
+    //"matrix": {
+    //    "omniscidbe4py": "",
+    //},
```

Comment:
why?

Reply:
omniscidbe4py already in `env_omnisci.yml` file that is used to create a conda environment. `conda install ...` command is called by ASV itself.

Reply:
then remove the section, what's the point of keeping it commented?..

Reply:
To know which fields can be changed if necessary (without looking for the documentation)

Reply:
If removed, then all unused fields. What do you think?

Reply:
I don't have a strong preference in this particular case, but usually I don't like comments being used instead of documentation for some third party stuff. So I would prefer we remove all stuff that isn't used.